src/test/java/uk/gov/hmcts/reform/coh/service/utils/QuestionDeadlineUtilsTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.coh.service.utils;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.coh.domain.Question;
+import uk.gov.hmcts.reform.coh.domain.QuestionState;
+import uk.gov.hmcts.reform.coh.service.QuestionStateService;
+import uk.gov.hmcts.reform.coh.states.QuestionStates;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+public class QuestionDeadlineUtilsTest {
+
+    @Mock
+    private QuestionStateService questionStateService;
+
+    @InjectMocks
+    private QuestionDeadlineUtils questionDeadlineUtils;
+
```

Comment:
You can use QuestionStateUtils to reduce some of the set up of QuestionState entities